guides/issuing-assets.md

```diff
@@ -408,10 +408,20 @@ System.out.println(trusted ? "Trusted :)" : "Not trusted :(");
 </code-example>
 
 
+## Provide Liquidity for your Asset
+
+When someone tries to buy or sell your asset on the decentralized exchange there needs to be a counterparty with whom they can trade. In other words, there needs to be enough [liquidity][liquidity] for your asset.
+
+This can be solved by using a [liquidity provider][market-maker]. This can be someone who is contracted to provide this service for a fee or as the asset issuer you can provide this liquidity by using a [market making bot][kelp].
```

Comment:
```suggestion
This can be solved by using a [liquidity provider][market-maker]. This can be someone who is contracted to provide this service for a fee, or as the asset issuer, you can provide this liquidity by using a [market making bot][kelp].
```